.github/workflows/release.yml

```diff
@@ -1,6 +1,7 @@
 # See https://github.com/containerd/nerdctl/blob/main/MAINTAINERS_GUIDE.md for how to make a release.
 name: Release
 on:
+  workflow_dispatch:
```

Comment:
?

Reply:
My bad, I was experimenting in my fork with GitHub actions. Did not realize it was this branch. Updated.